### ISSUE TITLE
SSL certificate for load balancer

### ISSUE THREAD

**Issue by pjsier** [CoderDojoChi/coderdojochi#490]
Given that there's a login on the site, it seems like a good idea for it to use SSL. Amazon Certificate Manager is free, and on Elastic Beanstalk you should be able to just add the certificate to the load balancer without actually changing the nginx configuration (apparently communication between the load balancer and the app server is over HTTP). 

More info here https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/configuring-https.html, and I'm happy to help with this

**Comment by karbassi** [comment]
@pjsier Yes, 100%. I tried this a few months ago and it works but our `assets` aren't working. Here's  our staging server where I tested. https://staging.coderdojochi.org/

**Comment by pjsier** [comment]
Hmm, not sure why on the staging site it's referencing the static files over http because the settings look right, and on staging the `STATIC_URL` setting shows up as HTTPS. It could have to do with `SITE_URL` being HTTP?

That part seems fixable though, and you can add the S3 bucket to a Cloudfront distro with a certificate through ACM set up to get around that.
